### ISSUE TITLE
Aria attribute aria-activedescendants has incorrect values in hierarchical grid, after selecting a child grid

### ISSUE THREAD

**Issue by dkamburov** [IgniteUI/igniteui-angular#10957]
## Description  
Aria attribute aria-activedescendants has incorrect values in hierarchical grid, after selecting a child grid

 * igniteui-angular version: 
 * browser: 

## Steps to reproduce  

1. Open the following sample - https://www.infragistics.com/angular-demos-lob/hierarchical-grid/hierarchical-grid-resizing it can also be tested with any of the hierarchical grid samples
2. Expand a row
3. Select a cell from a child grid
4. Inspect the aria-activedescendants for the root grid or for other child grid, than the currently active one.

## Result  
There are incorrect value for the aria attribute `aria-activedescendant="igx-hierarchical-grid-1_null_undefined`

## Expected result  
The aria attribute `aria-activedescendant` should have correct values for all child and root grids inside of a hierarchical grid

## Attachments  
![image](https://user-images.githubusercontent.com/1182001/152838288-e80890c9-e66d-4f86-8ee8-89ac25f82aa4.png)

**Comment by GibletJr** [comment]
Is there any ETA IGX can share on when this bug will be resolved?

**Comment by dkamburov** [comment]
@GibletJr We'll try to take a look on it by the end of the week
